src/main/java/com/connexta/store/adaptors/StorageAdaptorRetrieveResponse.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store.adaptors;
+
+import java.io.InputStream;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+import org.springframework.http.MediaType;
+
+@Data
+public class StorageAdaptorRetrieveResponse {
```

Comment:
Similar observation about naming this class.This class models an S3Object. It is very much like a File, but has a richer set of metadata than just filename. A good name might be DataObject, IonOjbect, or Blob (Binary Large OBject).